onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1164,6 +1164,11 @@ def _where_input_wrangler(
         matcher=lambda sample: any(dim == 0 for dim in sample.input.shape),
         reason="fixme: Logic not implemented for size 0 inputs in op.Reshape",
     ),
+    TorchLibOpInfo(
+        "unfold",
+        core_ops.aten_unfold,
+        trace_only=True,
+    ),
```

Comment:
```suggestion
    TorchLibOpInfo("unfold", core_ops.aten_unfold, trace_only=True),
```

nit: can be more compact

Reply:
changed